server/integration/infinispan/src/main/java/org/jboss/as/clustering/infinispan/cs/factory/DeployedCacheStoreFactory.java

```diff
@@ -31,12 +31,13 @@
          DeployedStoreConfiguration deployedConfiguration = (DeployedStoreConfiguration) cfg;
          DeployedCacheStoreMetadata deployedCacheStoreMetadata = null;
          try {
+            InfinispanLogger.ROOT_LOGGER.debug(String.format("Waiting for deployment of Custom Cache Store (%s).", deployedConfiguration.getCustomStoreClassName()));
             deployedCacheStoreMetadata = getPromise(deployedConfiguration.getCustomStoreClassName()).get(TIMEOUT_SECONDS, TimeUnit.SECONDS);
             return (T) deployedCacheStoreMetadata.getLoaderWriterRawInstance();
          } catch (InterruptedException | ExecutionException e) {
             throw new IllegalStateException("An error occurred while processing the deployment", e);
          } catch (TimeoutException e) {
-            InfinispanLogger.ROOT_LOGGER.error("Could not get Custom Cache Store metadata (" + deployedConfiguration.getCustomStoreClassName() + ") within given time.", e);
+            InfinispanLogger.ROOT_LOGGER.error(String.format("Waiting for deployment of Custom Cache Store (%s) timed out. Please check if this cache store is really present.", deployedConfiguration.getCustomStoreClassName()));
```

Comment:
Since you're modifying this piece - could you please move the error message to `InfinispanLogger.ROOT_LOGGER`?